### DISCUSSION TITLE
Standalone version example doesn't show MetaMask and Coinbase wallets

### DISCUSSION THREAD

**Discussion by dnmintroad** [WalletConnect/walletconnect-monorepo#1716] [Developer Support]
Hi! I would love some help. How do I setup Web3Modal to be able to connect MetaMask and Coinbase wallets?

I'm trying to setup a standalone version of web3modal but after setting up the example: https://docs.walletconnect.com/2.0/web3modal/standalone/installation

**Comment by Eyon42** [comment]
I don't know about Coibase wallet, but Metamask doesn't support v2 yet for the mobile app. That might be a reason why

**Comment by arein** [comment]
@dnmintroad web3modal standalone doesn't support Coinbase but web3modal general does. Why are you using standalone?
standalone does support MetaMask but you need to set `walletConnectVersion` to `'1'` as MetaMask will only support WalletConnect v2 starting in June as @Eyon42 pointed out.

**Comment by dnmintroad** [comment]
Got it. Thanks for the answers. I didn't end up using the standalone version - I interpreted the documentation wrong. Went with the React example. This discussion can be closed now. Thanks!
